### ISSUE TITLE
binary simlink name conflict with gitup.co

### ISSUE THREAD

**Issue by dva** [earwig/git-repo-updater#19]
[GitUp](http://gitup.co/) installs their Command Line Tools simlink over [git-repo-updater](https://github.com/earwig/git-repo-updater).

See build log [here](https://gist.github.com/dva/61f8c720739fd8f69217).

**Comment by earwig** [comment]
What can I honestly do here? We came first, as far as I can tell. I would suggest renaming this gitup, which you can do from setup.py, but that doesn't address the brew conflict. I would rather not rename this project, since it would mess with existing users' workflows.

**Comment by earwig** [comment]
Closing as I don't see anything for us to do here. At the risk of sounding petty and despite being a smaller project, we did come first by several years, and we do hold both the homebrew [formula](https://formulae.brew.sh/formula/gitup) and PyPI [package](https://pypi.org/project/gitup/) with the name "gitup".

If someone comes up with a way making the disambiguation clean that doesn't involve renaming the binary for everyone, I'm open.
